gensim/models/word2vec.py

```diff
@@ -334,8 +334,12 @@ def score_sentence_cbow(model, sentence, work=None, neu1=None):
 
 try:
     from gensim.models.word2vec_multistream import train_epoch_sg, train_epoch_cbow
+
+    MULTISTREAM_VERSION = 1
```

Comment:
constant should be in `gensim.models.word2vec_multistream.MULTISTREAM_VERSION`, and imported here